refgram/syntax/index.html

```diff
@@ -629,7 +629,7 @@ <h4 id="object_incorporation_pattern"><a href="#object_incorporation_pattern">Ob
 <p>In the above example, <span class="glyphs">~baq</span> <span class="toaq">bâq</span> has its normal rising tone replaced by the hiatus tone. The equivalent sentence without object incorporation would be:</p>
```

Comment:
This **bâq** was left untranscribed:
```suggestion
<p>In the above example, <span class="glyphs">󱚲󱛌󱚺󱛂</span> <span class="toaq">bâq</span> has its normal rising tone replaced by the hiatus tone. The equivalent sentence without object incorporation would be:</p>
```